kytos/templates/kytos.conf.template

```diff
@@ -14,8 +14,8 @@ pidfile = {{ prefix }}/var/run/kytos/kytosd.pid
 # This controller can be started in two modes: 'daemon' mode or 'interactive'
 # mode. On daemon mode, the process will detach from terminal when starts,
 # running in background. When running on 'interactive' mode, you will receive a
-# console right after the controller starts. Default is 'interactive' mode.
-daemon = False
+# console right after the controller starts. Default is 'daemon' mode.
+daemon = True
```

Comment:
I suggest leaving the default value commented - it's not needed, as it's already the default in the code.
```suggestion
# daemon = True
```

Reply:
It seems this causes many tests to fail, as shown in tox. The tests failing should have nothing to do with the changes in kytosd.py.

I believe it should not be commented out.

https://travis-ci.com/github/kytos/kytos/builds/191737058